### PR TITLE
Gradientbar

### PR DESCRIPTION
**Pull request by AndresFMoya**
This project consists of building a webpage using images as a background and adding gradients to elements. The goal is to make a clone of old an old version of the Apple website webpage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+                    placeholder="Search"
+                  />
+                </form>
+              </li>
+            </ul>
+          </nav>
+        </div>
+
+        <div class="title-images">
+          <img
```
Comment (mosaaleb) [inline index.html]: The images inside `title-images` should be clickable. They are actually links.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+              class="title-img-small"
+              src="images/cta_your_verse_27pt_2x.png"
+              alt="Watch the “Your Verse” film and explore the stories"
+          /></a>
+        </div>
+      </div>
+      <!-- Section with iphone models images -->
+      <div class="iphonemodels-section">
+        <ul class="iphonemodels-list">
+          <li class="iphonemodel">
```
Comment (mosaaleb) [inline index.html]: Also images inside `iphonemodels-list` are links.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="style/reset.css" />
+    <link rel="stylesheet" href="style/main.css" />
+    <title>Apple</title>
+  </head>
+
+  <body>
+    <div class="body-container">
```
Comment (mosaaleb) [inline index.html]: The main content should be wrapped around `<main>` tags. You can replace `div.body-container` with `main` to have more semantic tags.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @AndresFMoya

### Good job working in the project :clap:. You have already met most of the requirements, still there some requirements needs to be handled first before moving on. See my comments below.

There's also some redundant `id` selectors in the `nav` section. Let's remove them if they don't serve a purpose.

Please submit a new code review once you met these minor requirements.

Happy coding :computer:

**Comment by Oghenebrume50** [comment]
Hi, great work, just one requirement left to be met, according to the Microverse requirements, except for the main image, all other images on the site, should be passed as background images, that is the three images that are inscriptions, and the four images that show phone models in the footer should be passed as background images

File: index.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+            href="https://web.archive.org/web/20140228175622/http://www.apple.com/your-verse/#video-your-verse"
+            ><img
+              class="title-img-small"
+              src="images/cta_your_verse_27pt_2x.png"
+              alt="Watch the “Your Verse” film and explore the stories"
+          /></a>
+        </div>
+      </div>
+      <!-- Section with iphone models images -->
+      <div class="iphonemodels-section">
```
Comment (Oghenebrume50) [inline index.html:90]: and guys please this should be moved inside the  footer.

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]

**Comment by AndresFMoya** [comment]
@Oghenebrume50 The images are already set as backgrounds and modified the footer. I'll be pending. Thanks.

**Comment by mosaaleb** [comment]
reviewing

**Review by mosaaleb — APPROVED** [review]
## Hi @AndresFMoya :wave:

#### Good job :+1:. You can now merge into master

Good luck with the next projects.

#### Happy coding :computer:

**Pull request merged**